chart/templates/deployment.yaml

```diff
@@ -125,14 +121,11 @@ spec:
               cpu: "100m"
           env:
             # Note: in the case of remote execution plane we need to specify a list of CloudEvent types
-            {{- if .Values.remoteControlPlane.enabled }}
             - name: PUBSUB_TOPIC
               value: "{{ .Values.remoteControlPlane.topicSubscription }}"
-            {{- else }}
             # Note: In case of "normal" setup (within the same Kuberntes cluster as Keptn control-plane) it is enough to subscribe to sh.keptn.>
             - name: PUBSUB_TOPIC
               value: "sh.keptn.>" # all CloudEvent types
```

Comment:
I think we can remove the "generic" PUBSUB_TOPIC now. Also "sh.keptn.>" won't work in remote execution plane setups.